backup.sh

```diff
@@ -34,4 +34,7 @@ do
   fi
 done
 
-echo "Backup process finished at $(date "+%Y-%m-%d %H:%M:%S")"
\ No newline at end of file
+find $BACKUPDIR -type f -name '*.bak' -mtime +"$AGE_BACKUPS" -exec rm {} \;
```

Comment:
You should use the same naming scheme as for creating.

Look at...
`BAK_FILENAME=/backup/$CURRENT_DATE.$CURRENT_DB.bak`
[](
https://github.com/bbtsoftware/docker-mssql-backup/blob/f5aef0761c766f5cd64a1a9510be879fdbb4ebfa)/backup.sh#L14

I would use 
`find $BACKUPDIR -type f -name '*.$CURRENT_DB.bak' -mtime +"$AGE_BACKUPS" -exec rm {} \;`

... for each database (CURRENT_DB).